projects/plugins/jetpack/extensions/extended-blocks/core-site-logo/components/ai-toolbar-button.tsx

```diff
@@ -15,25 +15,32 @@ import type { ReactElement } from 'react';
  *
  * @param {object}   props              - The component props.
  * @param {Function} props.clickHandler - The handler for the click event.
+ * @param {boolean}  props.buttonText   - Use text for the button face or not.
  * @return {ReactElement} The toolbar button.
  */
 export default function AiToolbarButton( {
 	clickHandler,
+	buttonText = false,
```

Comment:
No need to change now, but since it is a boolean flag, the `buttonText` name can be misleading, as if you can provide the text for the button.

My suggestion is `showButtonText`.